ruma-signatures/src/functions.rs

```diff
@@ -506,11 +510,11 @@ where
     Ok(())
 }
 
-/// Uses a set of public keys to verify a signed event.
+/// Verifies that the signed event contains all the required valid signatures.
 ///
 /// Some room versions may require signatures from multiple homeservers, so this function takes a
-/// map from servers to sets of public keys. For each homeserver present in the map, this function
-/// will require a valid signature. All known public keys for a homeserver should be provided. The
+/// map from servers to sets of public keys. For each required homeserver, this function will
+/// require a valid signature. All known public keys for a homeserver should be provided. The
```

Comment:
Unrelated to your change, but I find the wording "will require" kind of weird. Do you agree it could just be "requires"?

Reply:
Or maybe fully rewrite to sth. like:

> Signatures are verified for each required homeserver.